boards/native/native_sim/native_sim.yaml

```diff
@@ -15,6 +15,7 @@ supported:
   - eeprom
   - netif:eth
   - usb_device
+  - usbd
```

Comment:
also for native_posix?

Reply:
Actually it is not correct, there is not specific driver for native_sim or native_posix. I will try to rework it.